src/js/profile/wearable/widget/wearable/Selector.js

```diff
@@ -1809,18 +1809,26 @@
 			/**
 			 * Remove item on specific layer
 			 * @method removeItem
-			 * @param {number} index
+			 * @param {number | HTMLElement} arg1 index of element to remove or HTML element to remove
```

Comment:
Please consider add new method removeItemByElement instead increase the method complexity.